test/integration/setup.js

```diff
@@ -431,26 +442,42 @@ async function setup () {
     ethToDaiRate,
   };
 
-  this.contractType = contractType;
 
-  const kycAuthSigner = '';
-  await enrollMember(this.contracts, members, kycAuthSigner);
+  this.contractType = contractType;
 
-  const signers = await ethers.getSigners();
   this.withEthers = web3ToEthers(this, signers);
 
+  await enrollMember(this.contracts, ethersAccounts.members, ethersAccounts.defaultSender);
+
+  const DEFAULT_POOL_FEE = '5'
```

Comment:
Could you check to see if your editor is using our `.eslintrc`? It's best to avoid unnecessary changes in the future when someone will auto-fix this on save.